src/app/shared/pipes/weather-icon.pipe.ts

```diff
@@ -8,22 +8,24 @@ export class WeatherIconPipe implements PipeTransform {
 	transform(value: string, args?: any): any {
 		let icons: { [icon: string]: string } = {
 			// From forecast.io docuemtnation
-			'clear-day'          : 'wi-day-sunny',
-			'clear-night'        : 'wi-night-clear',
-			'rain'               : 'wi-rain',
-			'snow'               : 'wi-snow',
-			'sleet'              : 'wi-sleet',
-			'wind'               : 'wi-cloudy-windy',
-			'fog'                : 'wi-fog',
-			'cloudy'             : 'wi-cloudy',
-			'partly-cloudy-day'  : 'wi-day-cloudy',
-			'partly-cloudy-night': 'wi-night-alt-cloudy',
-			// Forecast said maybe in future?
-			'hail'               : 'wi-hail',
-			'thunderstorm'       : 'wi-thunderstorm',
-			'tornado'            : 'wi-tornado',
-			// Default icon to fall back to
-			'default'            : 'wi-train'
+			'01d': 'wi-day-sunny',
+			'02d': 'wi-day-cloudy',
+			'03d': 'wi-cloud',
+			'04d': 'wi-cloudy',
+			'09d': 'wi-showers',
+			'10d': 'wi-day-rain',
+			'11d': 'wi-storm-showers',
+			'13d': 'wi-snow',
+			'50d': 'wi-fog',
+			'01n': 'wi-night-clear',
+			'02n': 'wi-night-alt-cloudy',
+			'03n': 'wi-cloud',
+			'04n': 'wi-cloudy',
+			'09n': 'wi-showers',
+			'10n': 'wi-night-rain',
+			'11n': 'wi-storm-showers',
+			'13n': 'wi-snow',
+			'50n': 'wi-fog'
 		};
 
 		return icons[value] || icons['default'];
```

Comment:
Do we need a default icon or does OpenWeather guarantee one?

Reply:
good point